app/policies/volunteer_policy.rb

```diff
@@ -1,15 +1,29 @@
 class VolunteerPolicy < ApplicationPolicy
-  alias_method :index?,           :superadmin?
-  alias_method :new?,             :superadmin?
-  alias_method :create?,          :superadmin?
+  class Scope < ApplicationScope
+    def resolve
+      return all if superadmin?
+      seeking_clients if department_manager?
+    end
+
+    def seeking_clients
+      return scope.seeking_clients_will_take_more if superadmin?
```

Comment:
If this def is called `if department_manager?` how is this line going to be called?

Reply:
If I understand correctly this is possible through `VolunteersController#seeking_clients`